components/LinkGroup/README.md

```diff
@@ -0,0 +1,154 @@
+# Link Group
```

Comment:
This documentation is titled link group, but a quick read tells me it's only talking about the Link itself, and nothing about link groups.